project/Build.scala

```diff
@@ -11,22 +11,22 @@ object Build extends AutoPlugin {
 
   object autoImport {
     val org = "com.sksamuel.elastic4s"
-    val AkkaVersion = "2.4.17"
-    val CatsVersion = "0.9.0"
-    val CirceVersion = "0.7.1"
+    val AkkaVersion = "2.5.27"
+    val CatsVersion = "2.0.0"
+    val CirceVersion = "0.12.0-M3"
```

Comment:
another pr adding 2.13 support to 6.5 handles this differently:
https://github.com/sksamuel/elastic4s/pull/1874/files (search for circeversion)

I could try doing the same if you think its important.

Reply:
This is fine.